validator/lib/readTileset.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+var fs = require('fs-extra');
+var Promise = require('bluebird');
+
+var fsReadFile = Promise.promisify(fs.readJson);
+
+function readTileset(filePath) {
+    return fsReadFile(filePath)
```

Comment:
`fsReadFile` returns a Buffer, but you are assuming it's a JSON object. It's good to have the buffer though because then you can check if it's gzipped. If it is gzipped you will need to decode the buffer, convert to a string, and then call `JSON.parse`. If it isn't gzipped, you can skip the decode step.